src/components/elements/MapPolygonPanel/MapMenuPanelItem.tsx

```diff
@@ -39,6 +39,7 @@ export interface MapMenuPanelItemProps extends DetailedHTMLProps<HTMLAttributes<
   primary_uuid?: string;
   isCollapsed?: boolean;
   validationStatus?: string;
+  isAdmin?: boolean;
```

Comment:
No need to open a new PR to change this, but for future reference: it's usually cleaner to use the hook in each component than to increase the props that are being passed around.